components/molecule/photoUploader/src/DragNotification.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import AtomIcon, {ATOM_ICON_SIZES} from '@s-ui/react-atom-icon'
+import MoleculeNotification, {
+  BRDS_SIZE
+} from '@s-ui/react-molecule-notification'
+
+const MOLECULE_NOTIFICATION_TYPE = 'error'
+const MOLECULE_NOTIFICATION_AUTOCLOSE = 'manual'
+
+const DragNotification = ({
+  icon,
+  onCloseCallback = () => {},
+  show = false,
+  text
+}) => {
+  return (
+    <>
+      {Boolean(show) && (
```

Comment:
It's confusing to me, show is already a `boolean`. Why do we need to do this? Am I missing any information?

Reply:
Nop, we're improving it! `show` is always a Boolean, there's no need to _bool_ it.